### ISSUE TITLE
[OSX] Right click options not working. 

### ISSUE THREAD

**Issue by jottr** [jiahaog/nativefier#95]
When selecting text, the right-click menu to copy text does not appear on osx. 
Might this be related to https://github.com/jiahaog/nativefier/issues/43?

```
$ nativefier --version
6.6.1

$ node --version
v5.4.1

$ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.11.2
BuildVersion:   15C50
```

**Comment by jiahaog** [comment]
Hi, that feature is not implemented and it's kind of low priority for me, as users can use the application menu to do so.

**Comment by lardissone** [comment]
I second this. It's required for copy images or copy URL on a link. Not only copy text that you can do by selecting and using menu action.

**Comment by Grawl** [comment]
Had same problem, update fixed it.

**Comment by sindresorhus** [comment]
You might find [electron-context-menu](https://github.com/sindresorhus/electron-context-menu) useful for this.

**Comment by jiahaog** [comment]
@sindresorhus Thank you for this, I'll look into implementing it

**Comment by danimalweb** [comment]
This would be great, did anyone implement it?

**Comment by bacongravy** [comment]
I believe this issue would be resolved by PR #579, which adds the electron-context-menu as a dependency of nativefier.
